app/pandas.ipynb

```diff
@@ -435,9 +550,7 @@
    "cell_type": "code",
    "execution_count": null,
    "outputs": [],
-   "source": [
-    "# filter s2 here"
-   ],
+   "source": "odd_s2 = s2[s2 % 2 != 0]",
```

Comment:
The filtering of odd numbers from s2 is defined but not executed. You should print the result to verify the operation.